src/extra/treemacs-all-the-icons.el

```diff
@@ -120,155 +114,107 @@
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "ruby-alt" :v-adjust 0 :face 'all-the-icons-red) treemacs-all-the-icons-tab) :extensions ("rb") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-fileicon "reason" :v-adjust 0 :face 'all-the-icons-red) treemacs-all-the-icons-tab) :extensions ("re" "rei") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "rust" :v-adjust 0 :face 'all-the-icons-red) treemacs-all-the-icons-tab) :extensions ("rs") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-fileicon "sbt" :v-adjust 0 :face 'all-the-icons-dsilver) treemacs-all-the-icons-tab) :extensions ("sbt") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "sass" :v-adjust 0 :face 'all-the-icons-lpink) treemacs-all-the-icons-tab) :extensions ("scss" "sass") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "scala" :v-adjust 0 :face 'all-the-icons-red) treemacs-all-the-icons-tab) :extensions ("scala") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-faicon "database" :v-adjust 0 :face 'all-the-icons-yellow) treemacs-all-the-icons-tab) :extensions ("sql") :fallback 'same-as-icon)
+    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-fileicon "racket" :v-adjust 0 :face 'all-the-icons-dred) treemacs-all-the-icons-tab) :extensions ("racket" "rktd" "rktl" "scrbl" "scribble" "plt") :fallback 'same-as-icon)
     (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-fileicon "sqlite" :v-adjust 0 :face 'all-the-icons-blue) treemacs-all-the-icons-tab) :extensions ("sqlite" "db3" "sqlite3") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "svg" :v-adjust 0 :face 'all-the-icons-orange) treemacs-all-the-icons-tab) :extensions ("svg") :fallback 'same-as-icon)
     (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-fileicon "swagger" :v-adjust 0 :face 'all-the-icons-green) treemacs-all-the-icons-tab) :extensions ("swagger") :fallback 'same-as-icon)
-    (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "swift" :v-adjust 0 :face 'all-the-icons-orange) treemacs-all-the-icons-tab) :extensions ("swift") :fallback 'same-as-icon)
     (treemacs-create-icon :icon (format "  %s%s" (all-the-icons-alltheicon "terminal" :v-adjust 0 :face 'all-the-icons-dgreen) treemacs-all-the-icons-tab)
-                          :extensions ("sh" "zsh" "zshrc" "zshenv" "fish" "zprofile" "zlogin" "zlogout" "bash"
-                                       "bash_profile" "bashrc" "bash_login" "profile" "bash_aliases")
+                          :extensions ("zshrc" "zshenv" "zprofile" "zlogin" "zlogout" "bash"
+				       "bash_profile" "bash_login" "profile" "bash_aliases")
```

Comment:
Wrong indentation.

Reply:
I think i removed these entries by mistake, during a test. I have added them back in.